querybook/webapp/components/DataDocCell/DataDocCell.tsx

```diff
@@ -80,6 +92,60 @@ export const DataDocCell: React.FunctionComponent<IDataDocCellProps> =
                 fullScreenCellIndex,
             } = useContext(DataDocContext);
 
+            let queryCells = useQueryCells(docId);
+            const queryTitle = queryCells[index].meta.title;
+            const title =
```

Comment:
i felt like duplicating cell title logic for the button below doesn't seem to be worth it, we can just have the title as `Run all cells below`